rust_src/src/remacs_sys.rs

```diff
@@ -127,6 +127,15 @@ extern "C" {
         dest: Lisp_Object,
         op: BoolVectorOp,
     ) -> Lisp_Object;
+    pub fn insert_from_string_1(
```

Comment:
Don't manually add things to remacs_sys. Instead, copy it to lisp.h or another header. Bindgen will find it and export it.

Reply:
Makes sense.